mkosi/util.py

```diff
@@ -269,3 +269,16 @@ def mandatory_variable(name: str) -> str:
         return os.environ[name]
     except KeyError:
         die(f"${name} must be set in the environment")
+
+
+def dir_only_contains(dir: Path, tgt: PathString) -> bool:
```

Comment:
Shouldn't this be called something like `is_valid_kdir` or so?

`not dir.is_dir():` returing `True` does not mean that `dir` only contains `tgt`...